docs/03-exports/aliases-api.include

```diff
@@ -745,6 +745,11 @@
 .. |REJECTED_BY_SAMPLES_LIMIT| replace:: :cpp:enumerator:`REJECTED_BY_SAMPLES_LIMIT<eprosima::fastdds::dds::SampleRejectedStatusKind::REJECTED_BY_SAMPLES_LIMIT>`
 .. |REJECTED_BY_SAMPLES_PER_INSTANCE_LIMIT| replace:: :cpp:enumerator:`REJECTED_BY_SAMPLES_PER_INSTANCE_LIMIT<eprosima::fastdds::dds::SampleRejectedStatusKind::REJECTED_BY_SAMPLES_PER_INSTANCE_LIMIT>`
 
+.. |GuardCondition::set_trigger_value-api| replace:: :cpp:func:`set_trigger_value()<eprosima::fastdds::dds::GuardCondition::set_trigger_value>`
+.. |StatusCondition::get_status_changed-api| replace:: :cpp:func:`get_status_changed()<eprosima::fastdds::dds::StatusCondition::get_status_changed>`
```

Comment:
StatusCondition does not have any API named as this. `get_status_changes` is API from Entity class.